modules/api/docs/LorisRESTAPI.md

```diff
@@ -263,6 +263,36 @@ will return a JSON object of the form:
 }
 ```
 
+#### 2.1.6 Single project dicoms tar files
```

Comment:
I don't think this file can be changed since the endpoint is available fro 0.0.4-dev only.